recipes/infomaniak-mail/webview.js

```diff
@@ -1,8 +1,25 @@
 module.exports = Ferdium => {
   const getMessages = () => {
-    const count = document.querySelector('.ws-tree-node-badge');
-    const countText = count ? count.textContent : null;
-    Ferdium.setBadge(count && countText ? Number(countText) : 0);
+    // This selects the first folder (the inbox and reads its unread messages count)
+    const inboxField = document.querySelector('.ws-tree-node-content')
+    const inboxCountField = inboxField.querySelector('.ws-tree-node-badge');
+    const inboxCountText = inboxCountField ? inboxCountField.textContent : null;
+    const inboxCount = inboxCountField && inboxCountText ? Number(inboxCountText) : 0;
```

Comment:
For the condition, `inboxCountField` is not necessary to be checked, since if that was undefined or null, then the prevopus line would have automatically returned a null. So, checking only for `inboxCountText` is enough here. Also, please use `Ferdium.safeParseInt` instead of the `Number` constructor.